app/scripts/templates/connect_another_device.mustache

```diff
@@ -64,7 +64,7 @@
       {{/isFirefoxIos}}
     {{/canSignIn}}
     <div class="links">
-      <a href="/connect_another_device/why">{{#t}}Why is this required?{{/t}}</a>
+      <a data-flow-event="link.why" href="/connect_another_device/why">{{#t}}Why is this required?{{/t}}</a>
```

Comment:
But it's THE CLOUD! 😄

I see your point but I think we're good. At some point soon, we'll implement automated expiry for the flow data like we have for the activity events and that should address most of the performance concerns. As long as they're useful events, the more the merrier.